PowershellTools.Common/Debugging/DebugEngineConstants.cs

```diff
@@ -163,5 +163,37 @@ public static class DebugEngineConstants
         /// Cmdlet to get a user's credentials. Used for attaching remotely. The $null Credential parameter supresses unwanted output in the REPL window.
         /// </summary>
         public const string GetCredentialsCommand = "Get-Credential -Credential $null";
+
+        /// <summary>
+        /// File name that is shared by all PowerShell hosts for either of the all hosts profiles.
+        /// </summary>
+        public const string AllHostsProfileFileName = "Profile.ps1";
+
+        /// <summary>
+        /// File name for profiles specifically for PowerShell tools
+        /// </summary>
+        public const string PowerShellToolsProfileFileName = "PoshTools_profile.ps1";
+
+        /// <summary>
+        /// Names of the various PowerShell profiles, in the order that they should be loaded.
+        /// </summary>
+        public static readonly string[] PowerShellProfiles =

```

Comment:
Avoid having two independent lookup tables when they are both used only in conjunction.  See KeyValuePair<> or Tuple<>